pkg/tasks/delete_repository_snapshots.go

```
@@ -86,8 +86,21 @@ func (d *DeleteRepositorySnapshots) Run() error {
 	// If pulp client is deleted, the org never had a domain created
 	if config.PulpConfigured() && d.pulpClient != nil {
 		snaps, _ := d.fetchSnapshots()
+
+		// Remove the "/latest" distribution
+		latestPathIdent := fmt.Sprintf("%v/%v", d.payload.RepoConfigUUID, "latest")
+
+		// Do not throw an error if not found
+		latestDistro, _ := d.getPulpClient().FindDistributionByPath(d.ctx, latestPathIdent)
```

shouldn't ignore the err there

---

i'd at least log it at error level with enough detail to know where it occurred